pkgs/development/libraries/czmq/4.x.nix

```diff
@@ -9,6 +9,13 @@ stdenv.mkDerivation rec {
     sha256 = "12gbh57xnz2v82x1g80gv4bwapmyzl00lbin5ix3swyac8i7m340";
   };
 
+  patches = [
+    (fetchpatch {
+      url = https://patch-diff.githubusercontent.com/raw/zeromq/czmq/pull/1618.patch;
+      sha256 = "1dssy7k0fni6djail8rz0lk8p777158jvrqhgn500i636gkxaxhp";
+    })
+  ];
```

Comment:
Doesn't it apply to czmq 3?

Reply:
No. Had to be backported.
The patch attached to `czmq3` is the same as `czmq4`, just backported to apply.